### DISCUSSION TITLE
Change canvas position property

### DISCUSSION THREAD

**Discussion by xtherexuu** [matteobruni/tsparticles#5145] [Q&A] [answered]
How can I change the properties from the 'style' attribute in the react-particles library for the canvas, for example, from 'position: fixed' to 'position: absolute,' etc.? I'm asking because when I add the 'style' property to the <Particles /> component, the styles only change if they haven't been previously set explicitly. For example, 'visibility: hidden' works, but 'position: absolute' doesn't.

**Comment by matteobruni** [comment, marked as answer]
https://github.com/matteobruni/tsparticles/issues/1277#issuecomment-1008909977
